.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish strongbox-examples to the GitHub Package Registry
+on:
+  [ push ]
+jobs:
+  publish-hello-strongbox-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to GitHub Packages
+      - uses: actions/setup-node@v1
+        with:
+          # LTS version see https://nodejs.org/en/
+          node-version: '12.x'
+          registry-url: 'https://npm.pkg.github.com'
+          # Defaults to the user or organization that owns the workflow file
+          scope: '@strongbox'
+      - run: npm install
+        working-directory: 'hello-strongbox-npm'
+      - run: npm publish
+        working-directory: 'hello-strongbox-npm'
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Jobs for deploying examples of other layout providers can just be appended here e.g.

```
publish-hello-strongbox-maven
...
publish-hello-strongbox-gradle
...
etc
```